core/App/screens/WebDisplay.tsx

```diff
@@ -0,0 +1,45 @@
+import { StackScreenProps } from '@react-navigation/stack'
+import React from 'react'
+import { StyleSheet, SafeAreaView } from 'react-native'
+import { WebView, WebViewNavigation } from 'react-native-webview'
+
+import { HomeStackParams, Screens } from '../types/navigators'
+
+type WebDisplayProps = StackScreenProps<HomeStackParams, Screens.WebDisplay>
+
+const WebDisplay: React.FC<WebDisplayProps> = ({ navigation, route }) => {
+  if (!route?.params) {
+    throw new Error('WebDisplay route prams were not set properly')
+  }
+
+  const { destUrl, exitUrl } = route?.params
+
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    container2: {
+      flex: 1,
+    },
+  })
+
+  return (
+    <SafeAreaView style={styles.container2}>
```

Comment:
At some point I'd like to be consistent with which `SafeAreaView` we import. I've adopted importing from the `react-native-safe-area-view-context` package and specifying the edges prop to remove the top and bottom padding of being used in a `TabStack`.

Reply:
Fixed.